### PR TITLE
Language filters

### PR DESCRIPTION
**Pull request by frankyma**
Branched off of: https://github.com/SeedCompany/cord-api-v3/pull/1513
Filter on sensitivity, leastOfThese, signLanguage, and Dialect

### CODE REVIEW

**Review by michaelmarshall — DISMISSED** [review]

**Comment by frankyma** [comment]
@michaelmarshall I merged the base to master can you pls re-review?

File: src/components/language/dto/list-language.dto.ts [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
+  Sensitivity,
   SortablePaginationInput,
 } from '../../../common';
 import { Language } from './language.dto';
 
 @InputType()
-export abstract class LanguageFilters {}
+export abstract class LanguageFilters {
+  @Field(() => [Sensitivity], {
+    description: 'Only projects with these sensitivities',
```
Comment (CarsonF) [inline src/components/language/dto/list-language.dto.ts:15]:
```suggestion
    description: 'Only languages with these sensitivities',
```

File: src/components/language/query.helpers.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        ? q
+            .match(propMatch('isSignLanguage'))
+            .where({ isSignLanguage: { value: true } })
+        : q
+    )
+    .call((q) =>
+      filter.isDialect
+        ? q.match(propMatch('isDialect')).where({ isDialect: { value: true } })
+        : q
+    );
```
Comment (CarsonF) [inline src/components/language/query.helpers.ts:32]: Can't these be merged into a single where clause that would be more performant?

Comment (frankyma) [reply src/components/language/query.helpers.ts:32]: I'm not sure, I'm not that familiar with the where clause.  What's the query you're imagining?  I can't think of one that allows for us to match on multiple of these filters together.

Comment (CarsonF) [reply src/components/language/query.helpers.ts:32]:
```ts
.where({
  a: ...,
  b: ...,
})
```
aka
```sql
WHERE a AND b
```

Comment (frankyma) [reply src/components/language/query.helpers.ts]: Ok I think I got it to work, pushed on the latest commit

**Pull request merged**